commands/orchestrationcommands/stackresourcecommands/get.go

```diff
@@ -0,0 +1,96 @@
+package stackresourcecommands
+
+import (
+    "github.com/rackspace/rack/commandoptions"
+    "github.com/rackspace/rack/handler"
+    "github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+    "github.com/rackspace/rack/internal/github.com/fatih/structs"
+    "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stackresources"
+    "github.com/rackspace/rack/util"
+)
+
+var get = cli.Command{
+    Name: "get",
+    Usage: util.Usage(commandPrefix, "get", "--name <stackName> --id <stackID> --resource <resourceName>"),
+    Description: "Show data for specified resource",
+    Action: actionGet,
+    Flags: commandoptions.CommandFlags(flagsGet, keysGet),
+    BashComplete: func(c *cli.Context){
+        commandoptions.CompleteFlags(commandoptions.CommandFlags(flagsGet, keysGet))
+    },
+}
+
+func flagsGet() []cli.Flag {
+    return []cli.Flag{
+        cli.StringFlag{
+            Name: "name",
+            Usage: "[required] The stack name.",
+        },
+        cli.StringFlag{
+            Name: "id",
+            Usage: "[required] The stack id.",
+        },
+        cli.StringFlag{
+            Name: "resource",
+            Usage: "[required] The resource name.",
+        },
+    }
+}
+
+type paramsGet struct {
+    stackName string
+    stackID string
+    resourceName string
+}
+
+var keysGet = []string{"Name", "PhysicalID", "Type", "Status", "UpdatedTime"}
```

Comment:
Ah, I actually meant is there a way to simply list *all* those keys without listing them explicity?